packages/calypso-e2e/src/lib/pages/themes-detail-page.ts

```diff
@@ -20,4 +31,40 @@ export class ThemesDetailPage extends BaseContainer {
 		await this.page.click( selectors.demoPane );
 		await PreviewComponent.Expect( this.page );
 	}
+
+	/**
+	 * Activates the theme.
+	 *
+	 * If the optional parameter `keepModal` is set to true, the Thanks message modal will not
+	 * be dismissed.
+	 *
+	 * @param {boolean} [keepModal] Optional parameter that if set to true will keep the thanks modal open. Defaults to false.
+	 * @returns {Promise<void>} No return value.
+	 */
+	async activate( { keepModal = false }: { keepModal?: boolean } = {} ): Promise< void > {
+		await this.page.click( selectors.activateDesignButton );
+		await this.page.waitForSelector( selectors.activateModal );
```

Comment:
I read some suggestions from Bart recently that proposed taking out any waiting for elements that won't actually be interacted with directly to add more stability to the test.

In that vein, what do you think of taking out this extra wait for the activateModal? The only thing we really need is the button in the next line (at least it seems to me).

Reply:
Good spotting. Will remove in next push.